document/js-api/index.bs

```diff
@@ -950,7 +950,7 @@ enum ValueType {
   "f32",
   "f64",
   "externref",
-  "anyfunc",
+  "funcref",
```

Comment:
You should keep both, and test that `new Global({ type: "funcref" })` works.

Reply:
Internally, we only use the "core" types, and the IDL enum here is only a way to expose them to JS. `ToValueType` then maps both enum values to `[=funcref=]`, and `FromValueType` only creates the `"funcref"` value. (Those are both already correct in the PR.) Does that make sense?

Reply:
You are saying that this is the set of strings that is allowed to be used in a `GlobalType`, and not the internal type. Fair enough, I added `anyfunc` back. PTAL